src/page/Page.tsx

```diff
@@ -26,7 +26,7 @@ export const NeoPage = ({
   isLoaded = true, // Whether the page is loaded and the cards can be displayed.
   onPageLayoutUpdate = () => {}, // action to take when the page layout is updated.
 }) => {
-  const getReportIndex = (pagenumber: number, index: number) => {
+  const getReportIndex = (pagenumber: number, index: string) => {
```

Comment:
should now be `getReportId` i think? and change parameter `index` to `id`?

Reply:
Used only for the grid keys